core/src/main/java/org/gbif/pipelines/core/functions/interpretation/DayInterpreter.java

```diff
@@ -0,0 +1,58 @@
+package org.gbif.pipelines.core.functions.interpretation;
+
+import org.gbif.pipelines.core.functions.interpretation.error.Issue;
+import org.gbif.pipelines.core.functions.interpretation.error.IssueType;
+import org.gbif.pipelines.core.functions.interpretation.error.Lineage;
+import org.gbif.pipelines.core.functions.interpretation.error.LineageType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * interprets day and adds issues and lineages with it
+ */
+public class DayInterpreter implements Interpretable<String, Integer> {
+
+  private static final int min_day = 1;
```

Comment:
constants variables must be in uppercase according to the coding style: MIN_DAY and MAX_DAY